src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -513,4 +513,23 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="Unknown" xml:space="preserve">
     <value>&lt;Unknown&gt;</value>
   </data>
+  <data name="IntellisenseBuildFailedMessage" xml:space="preserve">
+    <value>Encountered some issues while loading the project. this will prevent us from providing some features such as full solution analysis for this project and any projects that depend on this project.</value>
+  </data>
+  <data name="IntellisenseBuildFailedTitle" xml:space="preserve">
+    <value>Project loading failed.</value>
+  </data>
+  <data name="IntellisenseBuildFailedDescription" xml:space="preserve">
+    <value>To see what caused the issue, please try below.
```

Comment:
I made the change in update 2 PR (https://github.com/dotnet/roslyn/pull/9017) fwlink (http://go.microsoft.com/fwlink/p/?LinkID=734719 - currently it point to http://www.bing.com) we need to create a web post for the link. (https://github.com/dotnet/roslyn/issues/9157)

...

but I am not sure why this is better than using our bing dev search. if we want it to be always shown, then, we can ask bing dev search team to always include our official web post for the error code.